src/test/java/edu/harvard/iq/dataverse/importer/metadata/mxrdr/CbfImporterIT.java

```diff
@@ -0,0 +1,117 @@
+package edu.harvard.iq.dataverse.importer.metadata.mxrdr;
+
+import edu.harvard.iq.dataverse.importer.metadata.ImporterData;
+import edu.harvard.iq.dataverse.importer.metadata.ImporterFieldKey;
+import edu.harvard.iq.dataverse.importer.metadata.ImporterFieldType;
+import edu.harvard.iq.dataverse.importer.metadata.ImporterRegistry;
+import edu.harvard.iq.dataverse.importer.metadata.ResultField;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.ResourceBundle;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CbfImporterIT {
```

Comment:
Test doesn't look like integration test. What is the pont to treat it like one?

Reply:
It is also testing CbfMetadataFields besides CbfImporter, otherwise it would be Unit testing.

Reply:
Still think that it should have postfix `Test`. In general next time let's try avoid such constructs and try to write real unit tests (it will force you naturally to write simpler/easier to understand code that do the same).

Reply:
Yeah I was having doubts about making CbfMetadataFields not injectable, but at the other hand I felt like it didn't really deserve to be a "service", so I felt like since it is tightly coupled with cbf it would be ok.